tensorflow/tools/dockerfiles/partials/ubuntu-devel.partial.Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG UBUNTU_VERSION=16.04
+FROM ubuntu:${UBUNTU_VERSION}
```

Comment:
Why not just a devel.partial.Dockerfile and remove these lines and pull from ubuntu.partial.Dockerfile?

Reply:
I made it like this to have symmetry with the nvidia images. `nvidia` and `nvidia-devel` *need* to be separate because some of the images they install are mutually exclusive. I think it could be confusing if the pattern was different between the CPU and GPU images, although its current state *is* somewhat redundant.

That is, the spec definitions would look like this, which would get confusing if I were reading it:

- cpu-devel:
   - ubuntu
   - ubuntu-devel
   - devel-stuff
- gpu-devel:
   - nvidia-devel
   - devel-stuff

I think it could go either way, so this could definitely be changed by anyone updating the Dockerfiles later (e.g. it could be `[ubuntu-base, ubuntu-devel, ...]`)